File: Modul 7/Soal 4/PRAK604-2210817220007-AISYA ANINDIA SEPTHA.c
#include <stdio.h>
#include <string.h>
int main()
{
int a, b, c, d, e, f;
e = 0;
f = 0;
char i[100], j[100];
scanf("%[^\n]%*c", &i);
scanf("%[^\n]%*c", &j);
printf("\n");
a = strlen(i);
b = strlen(j);
if(a != b)
    {printf("Panjang kalimat berbeda, pesan palsu\n");}
else
    {for(c=0; c<a; c++)
        {if (i[c] == ' ' && j[c] == ' ')
            {printf("");}
        else if(i[c] == j[c])
            {printf("*");
            e++;
            }
        else
            {printf("#");
            f++;
            }
        }
    printf("\n* = %d", e);
    printf("\n# = %d", f);
    if(c > d)
        {printf("\nPesan Asli\n");}
    else if(c < d)
        {printf("\nPesan Palsu\n");}
    }
return 0;
}

File: Modul 7/Soal 4/PRAK604-2210817220007-AISYA ANINDIA SEPTHA.py
a = input()
b = input()
c = 0
d = 0
print("")
if len(a) != len(b) :
    print("Panjang kalimat berbeda, pesan palsu")
else :
    for e in range (len(a)) :
        if " " in a[e] and " " in b[e] :
            continue
        if a[e] == b[e] :
            print("*", end='')
            c = c + 1
        else :
            print("#", end='') 
            d = d + 1
    print("\n* = ", c)
    print("# = ", d)
    if c >= d :
        print("Pesan Asli")
    else :
        print("Pesan Palsu")
print("")